src/components/student-dashboard/dashboard.cjsx

```diff
@@ -0,0 +1,80 @@
+React  = require 'react'
+BS     = require 'react-bootstrap'
+moment = require 'moment'
+_ = require 'underscore'
+DontForgetPanel = require './dont-forget-panel'
+EmptyPanel      = require './empty-panel'
+UpcomingPanel   = require './upcoming-panel'
+AllEventsByWeek = require './all-events-by-week'
+ThisWeekPanel   = require './this-week-panel'
+
+{StudentDashboardStore} = require '../../flux/student-dashboard'
+
+module.exports = React.createClass
+  displayName: 'StudentDashboard'
+
+  propTypes:
+    courseId: React.PropTypes.any.isRequired
+
+  contextTypes:
+    router: React.PropTypes.func
+
+  getInitialState: ->
+    selectedTabIndex: 1
+
+  viewFlightPath: ->
+    @context.router.transitionTo('viewGuide', {courseId: @props.courseId})
+
+  selectTab: (index) ->
+    this.setState(selectedTabIndex:index)
```

Comment:
nope, other than I thought I was writing JS for a second.  I thought coffee-lint would catch those, but looks like it's not configured to do so, perhaps we could add that?